linux/net/rina/dtp-ps-default.c

```diff
@@ -84,6 +84,7 @@ int
 default_closed_window(struct dtp_ps * ps, struct pdu * pdu)
```

Comment:
Could you clean unneeded comments in this function?
The if clause for rate_based is really needed?:
```
+	if(dtcp_rate_based_fctrl(dtcp_config_get(dtcp))) {
+		LOG_DBG("rbfc CWQ: %zd, lim: %u", cwq_size(cwq), max_len);
+	}
```
The changes I think are valid here are the check on ```max_len``` and the ASSERT on the overrun policy.